modules/drivers/gnss/src/impl/novatel_parser.cpp

```diff
@@ -155,6 +162,8 @@ class NovatelParser : public Parser {
   ::apollo::drivers::gnss::Imu _imu;
   ::apollo::drivers::gnss::Ins _ins;
   ::apollo::drivers::gnss::InsStat _ins_stat;
+  ::apollo::drivers::gnss::GnssEphemeris _gnss_ephemeris;
+  ::apollo::drivers::gnss::EpochObservation _gnss_observation;
 };
 
 Parser* Parser::create_novatel() { return new NovatelParser(); }
```

Comment:
can we use protobuf for the messages, and use the 'AdapaterManager' interface? It will be easier to parse, and easier for other modules to use your message.

Reply:
Gnss driver use catkin compile currently.